index.js

```diff
@@ -12,6 +12,10 @@ var main = function(verbose) {
   manager.run(sites, settings, outAdapter, verbose);
 };
 
+function notCovered() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['notCovered' is defined but never used.](https://www.codacy.com/app/kevin_1379/codacy/file/1525424504/issues/source?bid=3007997&fileBranchId=3009961#l15)